### DISCUSSION TITLE
Bulk import of aliases

### DISCUSSION THREAD

**Discussion by js522** [anonaddy/anonaddy#317] [Feature Requests]
Hi @willbrowningme 

Do you have any idea of when [Bulk import of aliases for custom domain](https://github.com/anonaddy/anonaddy/projects/1#card-55408038) might be available?

This is quite an important feature for me, particularly for contingency planning (if the anonaddy service were to suddenly go away for some reason, "Bus factor" etc).

Thanks.

**Comment by willbrowningme** [comment]
I'll try to add it as soon as I can but I have some other things I'm working on for AnonAddy at the moment.

You can already export all of the data for your aliases (at the bottom of the settings page) if you're worried about the service suddenly going away.
